src/get_started.md

```diff
@@ -1,9 +1,9 @@
 # Getting Started 
 
 
-## A First "chat" Example
+## A simple example
 
-The following complete `index.html` webxdc app example shows an input field and every input is show on all peers.
+The following `index.html` shows a complete webxdc app, with an input field shown on all peers.
```

Comment:
```suggestion
The following `index.html` shows a complete webxdc app, with an input field shown on all peers.
Every input is show on all peers.
```

i think, this information is important to get the whole idea.

Reply:
What does 'every' mean here? As there is only one.

Reply:
Ah okay, when you say 'input' you mean the result of the `input`? So 'data sent from the input is delivered to all peers' kind of thing?